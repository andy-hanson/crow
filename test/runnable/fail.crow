import
	bootstrap: fut, str, void

main fut nat(_ arr str) summon unsafe
	foo
	0 resolved

foo void() unsafe summon
	bar

bar void() unsafe summon
	fail: "FOO"
