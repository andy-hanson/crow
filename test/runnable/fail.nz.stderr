uncaught exception: FOO
	at get-backtrace
	at fail<void>
	at bar
	at foo
	at main
	at add-first-task.lambda0
	at 
	at call<fut<?r>>
